srcpkgs/setzer/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'setzer'
+pkgname=setzer
+version=0.3.2
+revision=1
+archs="i686 x86_64"
+wrksrc=Setzer-${version}
+#create_wrksrc=yes
+build_style=meson
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="gettext python3-gobject-devel gobject-introspection python3-xdg xdg-utils meson"
```

Comment:
meson dependency is detected automatically, you don't need to specify it

Reply:
python3-gobject-devel should be moved to makedepends. Are python3-xdg, xdg-utils and gobject-introspection really needed here? It builds fine without them.

Reply:
Meson was removed, about python3-xdg and the other I didn't test building without them, the Setzer README clearly says:

> Run the following command to install prerequisite Debian packages:
> ```
> apt-get install python3-gi gir1.2-gtk-3.0 gir1.2-gtksource-4 gir1.2-gspell-1 gir1.2-pango-1.0 gir1.2-poppler-0.18 gir1.2-webkit2-4.0 python3-xdg gettext xdg-utils
> ```

Are we sure they aren't nedeed? If so, I'll remove them

Reply:
The README doesn't differentiate between runtime dependencies and build dependencies. These packages are not needed for the build. Move them to `depends=` if they are needed for runtime, you will have to test this.

Reply:
For python programs, you want to watch the `do_build` output to be sure python isn't trying to get dependencies using `EasyInstall` stuff. If it is, the dependencies should go in `hostmakedepends`, since the host's python interpreter needs to be able to find them. These same packages, when necessary for runtime, should also go in `depends`.